src/Adapter/ContainerBuilder.php

```diff
@@ -38,35 +47,183 @@
 class ContainerBuilder
 {
     /**
-     * @param string $name
+     * @var ContainerInterface
+     */
+    private static $containers;
+
+    /**
+     * @var string
+     */
+    private $environment;
+
+    /**
+     * @var string
+     */
+    private $containerName;
+
+    /**
+     * @var string
+     */
+    private $containerClassName;
+
+    /**
+     * @var string
+     */
+    private $dumpFile;
+
+    /**
+     * @var ConfigCache
+     */
+    private $containerConfigCache;
+
+    /**
+     * @param string $containerName
      * @param bool $isDebug
      *
      * @return SfContainerBuilder
      *
      * @throws \Exception
      */
-    public static function getContainer($name, $isDebug)
+    public static function getContainer($containerName, $isDebug)
     {
-        $containerName = ucfirst($name) . 'Container';
-        $file = _PS_CACHE_DIR_ . "${containerName}.php";
+        if (!isset(self::$containers[$containerName]) || null === self::$containers[$containerName]) {
```

Comment:
`!isset` is the same as `null ===`
```suggestion
        if (!isset(self::$containers[$containerName])) {
```

Reply:
right!